ValueConverters/PathToImageValueConverter.cs

```diff
@@ -13,6 +14,10 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         {
             var path = (string)value;
 
+            // skip if the file does not exist.
+            if (!File.Exists(path))
+                return null;
```

Comment:
When this happens, what shows up for the user?  An empty box?

Reply:
it looks like the texture is 100% transparent.

Reply:
I think long term we should at the very least provide an image or draw some text saying that the texture doesn't exist.  So I can merge this now or we can wait and add that in.  Do you have a preference?

Reply:
I would like to implement it later.
my idea would be, since it happens mainly while opening packs, i would show it directly that commponents are missing.

Reply:
Thanks must have missed that :)